test_unstructured/partition/docx/test_docx.py

```diff
@@ -601,13 +615,8 @@ def test_it_considers_text_inside_shapes():
 # -- module-level fixtures -----------------------------------------------------------------------
 
 
-def example_doc_path(filename: str) -> str:
-    """String path to a file in the example-docs/ directory."""
-    return str(pathlib.Path(__file__).parent.parent.parent.parent / "example-docs" / filename)
```

Comment:
This now has a better implementation in `test_unstructured.unit_utils`, use that one instead.